test/basic-querying.test.js

```diff
@@ -341,8 +341,9 @@ describe('basic-querying', function() {
         should.not.exist(err);
         should.exist(users);
         users.should.have.property('length', 2);
-        users[0].name.should.equal('John Lennon');
-        users[1].name.should.equal('Paul McCartney');
+        var expectedNames = ['John Lennon', 'Paul McCartney'];
+        expectedNames.indexOf(users[0].name).should.be.above(-1);
```

Comment:
I guess we're checking the index is not -1... would it be `should.not.equal(-1)`?